sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -0,0 +1,206 @@
+            "n_y": ["Ny", "Integer", 32, "Number of mesh points in y"],
+            "n_z": ["Nz", "Integer", 32, "Number of mesh points in z"],
+            "flag_bc": ["Flagbc", "Integer", 1, "Field boundary condition flag: Currently must be set to 1 which corresponds to an open boundary condition."],
+            "x_rad": ["Xrad", "Float", 0.015, "Size of computational domain"],
+            "y_rad": ["Yrad", "Float", 0.015, "Size of computational domain"],
+            "perd_len": ["Perdlen", "Float", 45.0, "Size of computational domain"],
+            "flag_dist": ["FlagDist", "DistType", "Gaussian", "Type of the initial distribution."],
+            "r_start_flg": ["Rstartflag", "Integer", 0, "If restartflag = 1, restart the simulation from the previous check point. If restartflag = 0, start the simulation from the beginning."],
+            "n_emission": ["Nemission", "Integer", 400, "The number of numerical emission steps"],
+            "t_emission": ["Temission [sec]", "Float", 6.5e-12, "Temission Laser pulse emission time"],
+            "distribution_parameters": {
+                "sigx": ["sigx [m]", "Float", 0.0006],
+                "sigpx": ["sigpx", "Float", 0.0],
+                "muxpx": ["muxpx", "Float", 0.0],
+                "xscale": ["xscale", "Float", 1.0],
+                "pxscale": ["pxscale", "Float", 1.0],
+                "xmu1": ["xmu1 [m]", "Float", 0.0],
+                "xmu2": ["xmu2", "Float", 0.0],
+                "sigy": ["sigy [m]", "Float", 0.0006],
+                "sigpy": ["sigpy", "Float", 0.0],
+                "muxpy": ["muxpy", "Float", 0.0],
+                "yscale": ["yscale", "Float", 1.0],
+                "pyscale": ["pyscale", "Float", 1.0],
+                "ymu1": ["ymu1 [m]", "Float", 0.0],
+                "ymu2": ["ymu2", "Float", 0.0],
+                "sigz": ["sigz [m]", "Float", 1.27e-06],
+                "sigpz": ["sigpz", "Float", 0.0],
+                "muxpz": ["muxpz", "Float", 0.0],
+                "zscale": ["zscale", "Float", 1.0],
+                "pzscale": ["pzscale", "Float", 1.0],
+                "zmu1": ["zmu1 [m]", "Float", 0.0],
+                "zmu2": ["zmu2", "Float", 0.0]
+            },
+            "b_curr": ["Bcurr [A]", "Float", 0.7140000000000003, "Beam current in Amps."],
+            "b_k_energy": ["Bkenergy [eV]", "Float", 1.0, "Initial beam pseudo-kinetic energy in eV."],
+            "b_mass": ["Bmas [eV]", "Float", 511005.0, "Mass of the particles in eV"],
+            "b_charge": ["Bcharge", "Float", -1.0,"Particle charge in units of proton charge."],
+            "b_freq": ["Bfreq [Hz]", "Float", 2856000000.0, "Reference frequency in Hz"],
+            "t_ini": ["Tini", "Float", -3.249141278122655e-12, "Initial reference time in seconds."]
+        },
+        "_ELEMENT": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
```

Comment:
Add a "name" field to _ELEMENT, even though it isn't required in the generated output - that way items can be named within the standard lattice editor.

Also all elements have a length field "L" which is the first line in the input row.